runtime/wasm/src/host.rs

```diff
@@ -718,3 +719,16 @@ impl RuntimeHostTrait for RuntimeHost {
         .await
     }
 }
+
+impl PartialEq for RuntimeHost {
+    fn eq(&self, other: &Self) -> bool {
+        // mapping_request_sender, host_exports and host_metrics are operational structs not needed
+        // to define uniqueness; each runtime host should be for a unique data source.
+        self.data_source_name == other.data_source_name
```

Comment:
I wonder if it isn't already enough to just compare the data source names. I believe those are guaranteed to be unique. Probably good to not rely on that though.

Reply:
Based on my sleuthing around the database, I don't think name uniqueness is enforced for dynamic data sources (standard data source names _are_ unique.)